src/valour.js

```diff
@@ -121,6 +121,15 @@ class Valour {
     });
     this.runCallbacks(name);
   }
+
+  disposeForm(name) {
+    this.forms[name] = {};
+  }
+
+  removeValidation(formName, key) {
+    let form = this.getForm(formName);
+    delete form[key];
```

Comment:
I think I'd rather see this written as a new form object as opposed to mutating the existing form. So something like
```javascript
this.forms[name] = Object.keys(this.getForm(formName)).reduce((newForm, key) => ..., {})
```